python/hail/dataset.py

```diff
@@ -1034,6 +1034,73 @@ def count(self):
         return r._1(), r._2()
 
     @handle_py4j
+    @typecheck_method(ped=Pedigree,
+                      population_AF=strlike,
+                      min_GQ=integral,
+                      min_p=numeric,
+                      max_parent_AB=numeric,
+                      min_child_AB=numeric,
+                      min_depth_ratio=numeric)
+    def de_novo(self, ped, population_AF, min_GQ=20, min_p=0.05,
+                max_parent_AB=0.05, min_child_AB=0.20, min_depth_ratio=0.10):
+        """Call de novo variation from trio data.
+
+        **Examples**
+
+        Call de novo events and export them to a file:
+
+        >>> pedigree = Pedigree.read('data/myStudy.fam')
+        >>> priors = hc.import_table('data/gnomadFreq.tsv', impute=True).key_by('Variant')
+        >>> denovo_kt = (vds.annotate_variants_table(priors, root='va.gnomAD')
+        ...                  .de_novo(pedigree, 'va.gnomAD'))
+        >>> denovo_kt.export('denovo_calls.tsv')
+
+        **Notes**
+
```

Comment:
I think it's worth including a few sentences describing in simple terms what this method does. de novo and proband are technical terms that may be unfamiliar to some users and make it sound overcomplicated. Something like:

For each trio of father, mother, and child (proband), this method finds violations of Mendelian inheritance and evaluates the probability each violation occurred due to a de novo mutation in the child (rather than genotyping error).